File: El Mamoun/Lv 1/Problem 26.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
/*
 Problem 26: Write a program to print numbers from 1 to N
 */
namespace Problem_26
{
    internal class Program
    {
        
        static int ReadNumber()
        {
            Console.WriteLine("Please Enter a Number?: ");
            int Number = Convert.ToInt32(Console.ReadLine());

            return Number;
        }
        static void PrintNumberFrom1ToN(int Number)
        {
            for(int Counter = 1; Counter <= Number; Counter++)
            {
                Console.WriteLine(Counter);
            }    
        }
        static void Main(string[] args)
        {
            PrintNumberFrom1ToN(ReadNumber());


            Console.ReadKey();
        }



    }
}

File: El Mamoun/Lv 1/Problem 26.cpp
#include<iostream>
using namespace std;

/*
 Problem 26: Write a program to print numbers from 1 to N
 */

int ReadNumber()
{
	int number = 0;

	cout << "Please Enter a Number? ";
	cin >> number;

	return number;
}

void PrintNumberFrom1ToN(int Number)
{

	for (int Counter = 1; Counter <= Number; Counter++)
	{
		cout << Counter << endl;
	}
}

int main()
{

	PrintNumberFrom1ToN(ReadNumber());

	return 0;
}